examples/01-filter/image-fft.py

```diff
@@ -64,6 +64,7 @@
     theme=grey_theme,
     log_scale=True,
     text='Moon Landing Image FFT',
+    copy_mesh=True,  # don't overwrite scalars when plotting
```

Comment:
If this fixes the error then we have to rethink how we handle complexes. This would mean that plotting with default `copy_mesh=False` would interfere with RFFT stuff, right?

Reply:
It's more of the fact that when we add scalars in `add_mesh` we also activate them. Now the active scalars are no longer complex and any `fft` filters are angry about invalid active scalars.